Keywords.txt

```diff
@@ -1,3 +1,4 @@
+Tainted | The enemy in which this is inflicted on has double the chance to be critically striked, however you cannot critically strike any other enemy besides the Tainted enemy. | 85 | 220 | 55
```

Comment:
'Tainted' seems like a bad word to use here as a keyword, maybe something like 'focused' would be more clear?